### PR TITLE
docs: setup prep checklist and full/brief wizard guide

### PR DESCRIPTION
**Pull request by ZappoMan**
## Summary

Adds a **pre-flight guide** and **in-wizard readiness briefings** so adopters know what `npm run setup:full` will ask before (and during) each phase—especially for cloud CI and GitHub secret sync.

### Documentation

- **[docs/setup-prep-checklist.md](docs/setup-prep-checklist.md)** — local vs full-cloud tracks; per-phase “have ready / you will be asked”; one-time secrets; post-wizard work (Stripe, OAuth, signed cookies, Lighthouse).
- Wired through **QUICKSTART** (§6.2, github phase pointer), **README**, **docs index**, **setup menu** option (2), and **CONTRIBUTING** (update checklist when changing wizard prompts).
- Phase-specific doc sections: **AWS** ([pr-preview-setup.md](docs/pr-preview-setup.md)), **Expo/Google** ([MOBILE_BUILD_TESTING.md](docs/MOBILE_BUILD_TESTING.md)), **Supabase PAT** ([supabase-staging-production-setup.md](docs/supabase-staging-production-setup.md)).

### Wizard UX (`setup-full`)

- **Full vs brief guide** — default full in-terminal checklist; `--guide=brief` / `--brief-guide` or interactive choice on first run; resume (`--from=`) uses brief banner.
- **Press Enter** readiness briefings after confirming each phase: **AWS**, **Expo**, **Google**, **GitHub** (with expanded Supabase account-PAT explanation in the supabase phase).
- **GitHub sync safety** — shows target repo (`gh repo view` / `git origin`), secret/variable counts, Y/n confirm; **type `owner/repo` to confirm** when target is upstream `Artificer-Innovations/BeakerStack`; skips sync on non-TTY; optional `--github-repo=owner/name`.
- **Expo** — treats `eas init` “Cannot automatically write to dynamic config” as success when project ID is recoverable from CLI output or `.eas/project.json`; patches `app.config.js` accordingly.

## Adopter notes

- **Fork, then sync GitHub secrets** on your fork’s `origin`. Cloning upstream directly will warn before writing to the template repo—use **N** / `--skip-github` on upstream clones.
- Skipped **expo** / **google**? Set `MOBILE_ENABLED=false` at github sync or use `--skip-mobile`.
- Signed cookies and Stripe webhooks remain **post-wizard**; see prep checklist “After the wizard”.

## Test plan

- [ ] Read `docs/setup-prep-checklist.md` against `npm run setup:full -- --help` phase list
- [ ] `npm run setup:full -- --dry-run` — full guide shows external prerequisites; `--brief-guide` shows doc pointers only
- [ ] Confirm phase briefings: `--from=aws`, `--from=expo`, `--from=google`, `--from=github` (Enter gates + github target summary)
- [ ] On upstream clone: github phase shows template warning; cancel without typing repo name
- [ ] On fork clone: github phase shows fork target and Y/n confirm
- [ ] `node --test scripts/__tests__/setup-github-repo.test.mjs`
- [ ] Expo: `eas init` on dynamic `app.config.js` links project without false failure
- [ ] `npm run setup` menu option (2) links prep doc; QUICKSTART §6.2 / §6.4 links resolve

### CODE REVIEW

**Review by mei-artificerinnovations — COMMENTED** [review]
Solid UX improvement overall. The GitHub sync safety gate (repo detection → count summary → Y/n → exact-type-for-upstream) is well-designed, and the full/brief guide toggle is a clean way to serve both first-run and returning adopters.

Three issues below — one is a rendering bug, the other two are suggestions.

File: scripts/setup-full.mjs [hunk 1]
```diff
@@ -565,22 +730,80 @@ async function promptExpoTokenForGithub(rl, promptInput, acc, flags) {
...
+  logInfo('');
+  logInfo('── Expo access token for GitHub Actions ──');
+  logInfo('');
+  logInfo(
+    'CI cannot use your local `eas login` session — it needs a separate access token (EXPO_TOKEN).'
+  );
+  logInfo('');
+  logInfo('Create this token type (on the page that opens next):');
+  logInfo(
+    '  • **Access token** — expo.dev → Account settings → Access tokens → Create token'
```
Comment (mei-artificerinnovations) [inline scripts/setup-full.mjs]: **Bug:** Markdown `**bold**` syntax renders literally in a terminal — users will see `  • **Access token** — expo.dev…` with the asterisks visible. Other bullets in this same function use plain text. Should be:
```js
'  • Access token — expo.dev → Account settings → Access tokens → Create token'
```

File: scripts/lib/setup-github-repo.mjs [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+  if (!opts.interactive) {
+    logWarn(
+      'Non-interactive terminal: skipping GitHub secret sync (cannot confirm target repo). Use a TTY, or set secrets manually on your fork.'
+    );
+    return false;
+  }
+
+  if (isBeakerstackTemplateRepo(repo)) {
+    logWarn('');
```
Comment (mei-artificerinnovations) [inline scripts/lib/setup-github-repo.mjs]: **Nit:** `logWarn('')`  will route blank lines through the warn handler — if the logger prefixes warn output with `[WARN]` or a colour code it produces visual noise on an otherwise-empty line. `logGithubSyncTargetSummary` just above this function uses `logInfo('')`  for its blank lines. Suggest matching that convention here so the warning block separators stay clean:
```js
logInfo('')
logWarn('⚠  You are about to write CI secrets…')
// ...
logInfo('')
```

File: scripts/__tests__/setup-github-repo.test.mjs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import assert from 'node:assert/strict';
```
Comment (mei-artificerinnovations) [inline scripts/__tests__/setup-github-repo.test.mjs:1]: Good to have tests for the pure helpers. The most complex new function — `confirmGithubRepoSyncTarget` — currently has no coverage: it handles the template-repo exact-type gate, the non-TTY skip, the fork/non-fork message, and the Y/n path. Since it depends on `rl` and `ctx`, it can be exercised with a minimal fake readline (`{ question: async () => 'y' }`) and a log-accumulator. Worth adding at least the template confirmation and the non-interactive skip paths before merge.

File: scripts/lib/setup-github-repo.mjs [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    logWarn('No GitHub repository resolved; skipping secret sync.');
+    return false;
+  }
+
+  if (!opts.interactive) {
+    logWarn(
+      'Non-interactive terminal: skipping GitHub secret sync (cannot confirm target repo). Use a TTY, or set secrets manually on your fork.'
+    );
+    return false;
+  }
```
Comment (sarah-artificerinnovations) [inline scripts/lib/setup-github-repo.mjs:195]: Nit: when `!opts.interactive` the log says "cannot confirm target repo — use a TTY" but there's no mention of `--github-repo=OWNER/NAME` as the escape hatch for headless environments. Adopters running in CI with a custom wrapper would benefit from that pointer:

```suggestion
    logWarn(
      'Non-interactive terminal: skipping GitHub secret sync (cannot confirm target repo). Use a TTY, pass --github-repo=OWNER/NAME to bypass confirmation, or set secrets manually on your fork.'
    );
```

File: scripts/lib/setup-expo-eas.mjs [hunk 5]
```diff
@@ -81,31 +84,49 @@ export async function clearTemplateEasLinkageFromMobileApp(ctx) {
...
   const tid = templateId.toLowerCase();
-  const nonTemplate = found.filter((id) => id.toLowerCase() !== tid);
+  const nonTemplate = found.filter(id => id.toLowerCase() !== tid);
   if (nonTemplate.length) {
     return nonTemplate[nonTemplate.length - 1];
   }
   return '';
 }
 
+/**
```
Comment (sarah-artificerinnovations) [inline scripts/lib/setup-expo-eas.mjs:102]: Nit: future readers will wonder why eas init ever exits 1 in the success path. A one-liner comment on the function would save them the archaeology:

```suggestion
/**
 * EAS init often exits 1 for app.config.js repos after printing the project id to add manually.
 * Dynamic config (app.config.js) repos cannot be patched by eas init automatically — it prints
 * the project id and exits non-zero, expecting the developer to add it. We detect this so we
 * can apply the patch ourselves via integrateEasProjectIdForDynamicConfig.
 * @param {string} combined stdout + stderr from eas init
 */
```

File: docs/setup-prep-checklist.md [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+| **Signed-cookie preview access** | **No** — separate script after AWS stack | See below                                                      |
+| **Lighthouse CI**                | No                                       | [lighthouse-ci.md](lighthouse-ci.md) — `LHCI_GITHUB_APP_TOKEN` |
+| **Branch protection**            | No                                       | [branch-protection-setup.md](branch-protection-setup.md)       |
+
+### Signed-cookie access (optional)
+
+By default PR previews and staging are **public**. To gate them with CloudFront **signed cookies** (internal stakeholders click a bootstrap link from the PR comment):
+
+**Have ready:** AWS stack already deployed (`PR_PREVIEW_STACK_NAME`, `PR_PREVIEW_DOMAIN` from the `aws` phase); `aws`, `openssl`, `jq`, `gh` in PATH.
+
```
Comment (sarah-artificerinnovations) [inline docs/setup-prep-checklist.md:221]: Nit: "you would be asked to type the repo name to confirm" reads a bit distant. Consider "you will be asked" (indicative) to match the voice used throughout the rest of the doc — e.g. "You will be asked: …" in every other phase section.

**Review by sarah-artificerinnovations — APPROVED** [review]
## Review: setup prep checklist + full/brief wizard guide

**Overall: LGTM — approve with minor nits (none blocking).**

The three main additions here are all solid:

1. **`docs/setup-prep-checklist.md`** — comprehensive, phase-by-phase, clearly flags one-time secrets. The "My values (copy locally)" template at the bottom is a nice touch.
2. **`scripts/lib/setup-github-repo.mjs`** — the upstream template detection + type-to-confirm guard is exactly the right UX for a scenario where a confused adopter could accidentally push secrets to the public repo. Tests cover the core URL parsing cases.
3. **`easInitFailedDueToDynamicAppConfig`** — clean solution to the real EAS quirk where `eas init` exits 1 for dynamic config repos even after writing the project id. Switching to `runEasCapture` + checking `linkedOnDisk` as a fallback is more robust than the old TTY passthrough.

Three small nits below.

File: scripts/lib/setup-expo-eas.mjs [hunk 7]
```diff
@@ -81,31 +84,49 @@ export async function clearTemplateEasLinkageFromMobileApp(ctx) {
  * @param {string} [templateId]
  * @returns {string} first plausible EAS project UUID not equal to template, else ''
```
Comment (diego-artificerinnovations) [inline scripts/lib/setup-expo-eas.mjs]: nit: JSDoc says "first plausible EAS project UUID" but the implementation returns the **last** one (`nonTemplate[nonTemplate.length - 1]`). Returning last is probably intentional — EAS CLI puts the newly-created project ID near the end of its output. The docstring should say "last" to match:

```js
 * @returns {string} last plausible EAS project UUID not equal to template, else ''
```

File: scripts/lib/setup-github-repo.mjs [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+ * @param {{ logInfo: (s: string) => void; logWarn: (s: string) => void }} ctx
+ * @param {ReturnType<typeof resolveGhRepoContext>} ghCtx
+ * @param {{ dryRun: boolean; interactive: boolean }} opts
+ * @returns {Promise<boolean>}
+ */
+export async function confirmGithubRepoSyncTarget(rl, ctx, ghCtx, opts) {
+  const { logInfo, logWarn } = ctx;
+  const { repo, isFork, parentRepo } = ghCtx;
+  if (!repo) {
+    logWarn('No GitHub repository resolved; skipping secret sync.');
```
Comment (diego-artificerinnovations) [inline scripts/lib/setup-github-repo.mjs:180]: nit: the `if (!repo)` guard on lines 166–168 is unreachable — `setup-full.mjs` already returns early when `!ghCtx.repo` before calling this function. Harmless defensive coding, but it could mislead a future reader into thinking the function can be called without a resolved repo. Either remove the guard or add a brief comment marking it as intentional belt-and-suspenders.

File: scripts/__tests__/setup-github-repo.test.mjs [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    parseGithubOwnerRepoFromRemoteUrl('git@github.com:myorg/my-app.git'),
+    'myorg/my-app'
+  );
+});
+
+test('isBeakerstackTemplateRepo', () => {
+  assert.ok(isBeakerstackTemplateRepo(BEAKERSTACK_TEMPLATE_GITHUB_REPO));
+  assert.ok(isBeakerstackTemplateRepo('artificer-innovations/beakerstack'));
+  assert.ok(!isBeakerstackTemplateRepo('myuser/BeakerStack'));
+});
```
Comment (diego-artificerinnovations) [inline scripts/__tests__/setup-github-repo.test.mjs:58]: nit: the test suite covers SSH+HTTPS-with-.git+case-insensitive — good starts, but a few cheap cases are missing:

```js
// HTTPS without .git (the URL parser strips .git conditionally)
test("parseGithubOwnerRepoFromRemoteUrl https without .git", () => {
  assert.equal(
    parseGithubOwnerRepoFromRemoteUrl("https://github.com/myorg/my-app"),
    "myorg/my-app"
  );
});

// Empty / falsy input → empty string
test("parseGithubOwnerRepoFromRemoteUrl empty", () => {
  assert.equal(parseGithubOwnerRepoFromRemoteUrl(""), "");
  assert.equal(parseGithubOwnerRepoFromRemoteUrl(null), "");
});

// Non-GitHub hostname → empty string (protects isBeakerstackTemplateRepo)
test("parseGithubOwnerRepoFromRemoteUrl non-github", () => {
  assert.equal(
    parseGithubOwnerRepoFromRemoteUrl("https://gitlab.com/myorg/my-app.git"),
    ""
  );
});
```

The non-GitHub case in particular matters because `isBeakerstackTemplateRepo` relies on `parseGithubOwnerRepoFromRemoteUrl` being strict about the hostname.

**Review by diego-artificerinnovations — APPROVED** [review]
Solid UX improvement. The GitHub sync safety guard (type-to-confirm for upstream template, Y/n for forks, non-TTY skip) is the right call and correctly implemented. The per-phase readiness briefings are thorough without being overwhelming, and the prep checklist is an excellent addition — especially the "decide first" table and the "my values" scratchpad at the bottom.

Three nits below, none blocking.

File: scripts/lib/setup-github-repo.mjs [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+ * @param {{ secretCount: number; variableCount: number }} counts
+ */
+export function logGithubSyncTargetSummary(ctx, ghCtx, counts) {
+  const { logInfo, logWarn } = ctx;
+  const { repo, isFork, parentRepo, originRepo, originUrl } = ghCtx;
+  logInfo('');
+  logInfo('── GitHub sync target ──');
+  logInfo(`  Repository: ${repo || '(unknown)'}`);
+  if (originUrl) logInfo(`  git remote origin: ${originUrl}`);
+  if (originRepo && originRepo !== repo) {
```
Comment (rahul-artificerinnovations) [inline scripts/lib/setup-github-repo.mjs]: When `--github-repo` is passed, this path is taken and `gh repo view` is never called — but the warning on the next lines reads _"gh repo view resolved `<repo>`"_, which is factually wrong (the value came from the flag, not from `gh repo view`). It also fires for the intentional case where a developer deliberately overrides the target repo. Consider threading a `forced: boolean` through the returned context object so this warning can read _"`--github-repo` override targets `<repo>`, origin is `<originRepo>`"_ instead.

File: scripts/__tests__/setup-github-repo.test.mjs (hunk 9 above)
Comment (rahul-artificerinnovations) [inline scripts/__tests__/setup-github-repo.test.mjs:58]: The happy-path coverage is good. `parseGithubOwnerRepoFromRemoteUrl` has guards for unknown schemes, but they're untested — worth adding a few edge cases: empty string, a non-GitHub SSH URL, a URL without `.git` suffix. Low cost and catches any future regex tightening that accidentally breaks the fallback.

File: docs/setup-prep-checklist.md [hunk 11]
```diff
@@ -0,0 +1,279 @@
...
+Add `--enable-staging` to lock staging too. The script generates an RSA key pair, updates CloudFormation (`signed-cookies` mode), and sets GitHub secrets `CLOUDFRONT_SIGNING_KEY` + `CLOUDFRONT_SIGNING_KEY_ID`. Allow **5–10 minutes** for CloudFront to propagate.
+
+If you skip the script but add those secrets manually, the `github` phase may still prompt for them when syncing CI.
+
+**More:** [preview-access-control.md](preview-access-control.md)
+
+---
+
+## My values (copy locally — do not commit)
```
Comment (rahul-artificerinnovations) [inline docs/setup-prep-checklist.md:239]: "Copy locally" in the heading is slightly ambiguous for a new adopter — they might edit this committed file in place rather than creating a scratch file outside the repo. The top-of-file note says _"Do not commit filled-in values"_, which helps, but a nudge right at the template section would reinforce it: e.g. `> Copy to a file outside this repo (e.g. `~/beakerstack-setup.txt`) before filling in.`

**Review by rahul-artificerinnovations — APPROVED** [review]
Clean PR. The prep checklist fills a real gap (gathering credentials before hitting a blocking prompt is genuinely better UX), the phase-briefing gate is well-paced, and the `--github-repo` override + BeakerStack type-to-confirm guard are both the right call. Three targeted observations below — none are blockers.

File: scripts/lib/setup-github-repo.mjs (hunk 10 above)
Comment (rahul-artificerinnovations) [inline scripts/lib/setup-github-repo.mjs]: When `--github-repo` is passed, this path is taken and `gh repo view` is never called — but the warning on the next lines reads _"gh repo view resolved `<repo>`"_, which is factually wrong (the value came from the flag, not from `gh repo view`). It also fires for the intentional case where a developer deliberately overrides the target repo. Consider threading a `forced: boolean` through the returned context object so this warning can read _"`--github-repo` override targets `<repo>`, origin is `<originRepo>`"_ instead.

File: scripts/__tests__/setup-github-repo.test.mjs (hunk 9 above)
Comment (rahul-artificerinnovations) [inline scripts/__tests__/setup-github-repo.test.mjs:58]: The happy-path coverage is good. `parseGithubOwnerRepoFromRemoteUrl` has guards for unknown schemes, but they're untested — worth adding a few edge cases: empty string, a non-GitHub SSH URL, a URL without `.git` suffix. Low cost and catches any future regex tightening that accidentally breaks the fallback.

File: docs/setup-prep-checklist.md (hunk 11 above)
Comment (rahul-artificerinnovations) [inline docs/setup-prep-checklist.md:239]: "Copy locally" in the heading is slightly ambiguous for a new adopter — they might edit this committed file in place rather than creating a scratch file outside the repo. The top-of-file note says _"Do not commit filled-in values"_, which helps, but a nudge right at the template section would reinforce it: e.g. `> Copy to a file outside this repo (e.g. `~/beakerstack-setup.txt`) before filling in.`

**Review by rahul-artificerinnovations — APPROVED** [review]
Clean PR. The prep checklist fills a real gap (gathering credentials before hitting a blocking prompt is genuinely better UX), the phase-briefing gate is well-paced, and the `--github-repo` override + BeakerStack type-to-confirm guard are both the right call. Three targeted observations below — none are blockers.

File: scripts/setup-full.mjs (hunk 1 above)
Comment (ZappoMan) [reply scripts/setup-full.mjs]: Fixed in 0273b6b — removed the `**` markers so the Expo token bullet renders as plain terminal text.

File: scripts/lib/setup-github-repo.mjs (hunk 2 above)
Comment (ZappoMan) [reply scripts/lib/setup-github-repo.mjs]: Fixed — blank separators around the upstream warning now use `logInfo` so they are not prefixed with `[WARN]`.

File: scripts/__tests__/setup-github-repo.test.mjs (hunk 3 above)
Comment (ZappoMan) [reply scripts/__tests__/setup-github-repo.test.mjs:1]: Added tests in `setup-github-repo.test.mjs` for non-interactive skip, `--github-repo` override proceed, template type-to-confirm (reject + accept), Y/n happy path, and missing-repo guard.

File: scripts/lib/setup-github-repo.mjs (hunk 4 above)
Comment (ZappoMan) [reply scripts/lib/setup-github-repo.mjs:195]: Updated the non-interactive message and implemented the escape hatch: when `--github-repo=OWNER/NAME` is set (and the target is not the upstream template), sync proceeds without a TTY prompt. Template repo still requires interactive type-to-confirm.

File: scripts/lib/setup-expo-eas.mjs (hunk 5 above)
Comment (ZappoMan) [reply scripts/lib/setup-expo-eas.mjs:102]: Expanded the JSDoc on `easInitFailedDueToDynamicAppConfig` to document the exit-1 / dynamic-config behavior and why we patch via `integrateEasProjectIdForDynamicConfig`.

File: docs/setup-prep-checklist.md (hunk 6 above)
Comment (ZappoMan) [reply docs/setup-prep-checklist.md:221]: Changed to "you will be asked" in the github section for consistent voice.

File: scripts/lib/setup-expo-eas.mjs (hunk 7 above)
Comment (ZappoMan) [reply scripts/lib/setup-expo-eas.mjs]: JSDoc now says "last plausible" to match `nonTemplate[nonTemplate.length - 1]` (EAS tends to print the new project id near the end of output).

File: scripts/lib/setup-github-repo.mjs (hunk 8 above)
Comment (ZappoMan) [reply scripts/lib/setup-github-repo.mjs:180]: Kept the guard with a short comment marking it as belt-and-suspenders for direct callers/tests (setup-full already bails when `!repo`).

File: scripts/__tests__/setup-github-repo.test.mjs (hunk 9 above)
Comment (ZappoMan) [reply scripts/__tests__/setup-github-repo.test.mjs:58]: Added HTTPS-without-.git, empty/null, and non-GitHub hostname cases to the test suite.

File: scripts/lib/setup-github-repo.mjs (hunk 10 above)
Comment (ZappoMan) [reply scripts/lib/setup-github-repo.mjs]: Added `repoFromOverride` on the context object; the summary now says `--github-repo override` instead of "gh repo view resolved" when the flag is used.

File: docs/setup-prep-checklist.md (hunk 11 above)
Comment (ZappoMan) [reply docs/setup-prep-checklist.md:239]: Added a blockquote under "My values" pointing adopters to copy the scratchpad outside the repo (e.g. `~/beakerstack-setup.txt`).

**Comment by sarah-artificerinnovations** [comment]
All three nits resolved — non-interactive warning now mentions `--github-repo` as the escape hatch, `easInitFailedDueToDynamicAppConfig` JSDoc explains the dynamic-config exit-1 case, and "would be asked" → "will be asked" throughout. Good to merge.

**Review by diego-artificerinnovations — APPROVED** [review]
All three addressed — re-approving.

- **JSDoc "first" → "last"**: fixed in `extractEasProjectIdFromCliOutput`.
- **Dead `!repo` guard**: comment added explaining it's intentional for direct callers/tests, and the new tests prove the point.
- **Test coverage**: fully expanded — HTTPS-without-.git, empty/null, non-GitHub hostname, plus thorough `confirmGithubRepoSyncTarget` and `logGithubSyncTargetSummary` coverage.

Bonus improvements appreciated: `repoFromOverride` flag + non-interactive path proceeding with `--github-repo` override is genuinely useful for CI, and swapping `logWarn('')` → `logInfo('')` for blank-line separators is the right semantic call.

**Review by rahul-artificerinnovations — APPROVED** [review]
All three observations addressed. `repoFromOverride` is properly threaded through context and the summary logger now branches the message correctly. Test coverage expanded substantially — edge cases for `parseGithubOwnerRepoFromRemoteUrl` (empty, null, non-GitHub, no-`.git`), the full `confirmGithubRepoSyncTarget` matrix, and a dedicated `logGithubSyncTargetSummary` test verifying the override message. The checklist callout block is clear. Good to merge.

File: scripts/__tests__/setup-github-repo.test.mjs (hunk 9 above)
Comment (ZappoMan) [reply scripts/__tests__/setup-github-repo.test.mjs:58]: Yes — addressed in 0273b6b with tests for empty/null, HTTPS without `.git`, and non-GitHub HTTPS (GitLab). Added `git@gitlab.com:…` SSH in a follow-up commit on this branch so all three cases you called out are covered.

**Review by mei-artificerinnovations — APPROVED** [review]
All three items addressed — markdown bold removed from terminal output, blank separator lines switched from `logWarn` to `logInfo`, and `confirmGithubRepoSyncTarget` tests added covering the template gate and non-interactive skip path. Approved.

File: scripts/lib/setup-github-repo.mjs (hunk 10 above)
Comment (ZappoMan) [reply scripts/lib/setup-github-repo.mjs]: Addressed in 0273b6b: added `repoFromOverride` on the gh context. When `--github-repo` is set, the summary logs `Target from --github-repo override (not gh repo view)` and only warns about origin mismatch with accurate wording (no false "gh repo view resolved").

**Review by mei-artificerinnovations — APPROVED** [review]
All three items addressed — markdown bold removed from terminal output, blank separator lines switched from `logWarn` to `logInfo`, and `confirmGithubRepoSyncTarget` tests added covering the template gate and non-interactive skip path. Approved.

File: scripts/__tests__/setup-github-repo.test.mjs (hunk 9 above)
Comment (ZappoMan) [reply scripts/__tests__/setup-github-repo.test.mjs:58]: Addressed in 0273b6b + f4d6ae7: tests for empty/null, HTTPS without `.git`, non-GitHub HTTPS (GitLab), and non-GitHub SSH (`git@gitlab.com:…` → empty string). See `scripts/__tests__/setup-github-repo.test.mjs`.

File: docs/setup-prep-checklist.md (hunk 11 above)
Comment (ZappoMan) [reply docs/setup-prep-checklist.md:239]: Addressed in 0273b6b: added a blockquote directly under "## My values" — copy to a file outside the repo (e.g. `~/beakerstack-setup.txt`) before filling in, in addition to the top-of-file do-not-commit note.
